test/integration/targets/connection_aws_ssm/runme.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Setup Environment
+ansible-playbook -c local aws_ssm_integration_test_setup.yml "$@"
+
+# Execute test cases only on Python 2.7
+if [ -e aws-env-vars.sh ]; then
+
+    # Export the AWS Keys
+    . ./aws-env-vars.sh
+
+    cd ../connection
+
+    # Execute Integration tests for Linux
+    INVENTORY=../connection_aws_ssm/inventory-linux.aws_ssm ./test.sh \
+        -e target_hosts=aws_ssm \
+        -e local_tmp=/tmp/ansible-local \
+        -e remote_tmp=/tmp/ansible-remote \
+        -e action_prefix= \
+        "$@" || true
+
+    # Execute Integration tests for Windows
+    INVENTORY=../connection_aws_ssm/inventory-windows.aws_ssm ./test.sh \
+        -e target_hosts=aws_ssm \
+        -e local_tmp=/tmp/ansible-local \
+        -e remote_tmp=c:/windows/temp/ansible-remote \
+        -e action_prefix=win_ \
+        "$@" || true
+
+    cd ../connection_aws_ssm
+
+    # Destroy Environment
+    ansible-playbook -c local aws_ssm_integration_test_teardown.yml "$@"
```

Comment:
Use an exit trap to run this when tests finish. Example:

https://github.com/ansible/ansible/blob/aa32164d15b76fbf836693efd31a42fab6e8d938/test/integration/targets/callback_default/runme.sh#L69

Reply:
Fixed in https://github.com/ansible/ansible/pull/49652/commits/d69abb2abcc0c00912ae786c9f2a81bf489073a7